core-tests/shared/src/test/scala/zio/prelude/AssertionSpec.scala

```diff
@@ -0,0 +1,20 @@
+package zio.prelude
+
+import zio.prelude.Assertion
+import zio.test.Assertion._
+import zio.test.{Assertion => _, _}
```

Comment:
I think you need to figure out a different way to do this, because Scala 2.11 doesn't like this :(
Perhaps importing things in `zio.test` explicitly?